docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+| CorsHeaderExistenceRequestAfterRoutingValidatorPlugin | Executes validations that need to be aware of the resolved route.               | Spryker\Glue\GlueHttp\Plugin\GlueStorefrontApiApplication                                                   |
+| ConventionResourceFilterPlugin                        | Filters resources for JSON API convention.                                      | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| ResourceRouteMatcherPlugin                            | Routes API requests using the `ResourceInterface`.                              | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| RequestResourceFilterPlugin                           | Filters resources by `GlueRequestTransfer` resource name.                       | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| JsonApiApiConventionPlugin                            | Allows the convention to follow the request flow.                               | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueApplication                                                   |
+| SparseFieldRequestBuilderPlugin                       | Extends `GlueRequestTransfer.sparseResources` with `GlueSparseResourceTransfer` | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| AttributesRequestBuilderPlugin                        | Extends `GlueRequestTransfer.attributes` with content attributes.               | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| RelationshipRequestBuilderPlugin                      | Extends `GlueRequestTransfer.included` with included fields.                    | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| PaginationRequestBuilderPlugin                        | Extends `GlueRequestTransfer.pagination` with `PaginationTransfer`.             | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| SortRequestBuilderPlugin                              | Extends `GlueRequestTransfer.sorting` with `SortTransfer`.                      | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| FilterFieldRequestBuilderPlugin                       | Extends `GlueRequestTransfer.filter` with `GlueFilterTransfer`.                 | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
```

Comment:
All of these should be saying `Builds `GlueRequestTransfer.*` from raw request data.`